### ISSUE TITLE
WCT failing on Firefox 47

### ISSUE THREAD

**Issue by gazal-k** [Polymer/web-component-tester#317]
Please check here for a build log:
https://travis-ci.org/gazal-k/paper-date-time-input/builds/136715687

The tests worked fine on Firefox 46

**Comment by nick-woodward** [comment]
The root of this issue seems to be related to SeleniumHQ/selenium#2110.

**Comment by rictic** [comment]
Yep. At the moment we're falling back to testing only in Firefox 46. We're exploring solutions.

**Comment by LM450N** [comment]
It has been fixed in the [release 47.0.1](https://www.mozilla.org/en-US/firefox/47.0.1/releasenotes/).
